flink-runtime/src/main/java/org/apache/flink/runtime/codegeneration/TemplateManager.java

```diff
@@ -41,10 +41,10 @@
 	// ------------------------------------------------------------------------
 	//                                   Constants
 	// ------------------------------------------------------------------------
-	public static final String RESOURCE_PATH  = "/Users/heytitle/projects/apache-flink/flink-runtime/resources";
-	public static final String TEMPLATE_PATH  = RESOURCE_PATH + "/templates";
+	public static final String RESOURCE_PATH  = createTemporaryDirectory();
+	public static final String TEMPLATE_PATH  = createSubFolder(RESOURCE_PATH , "templates");
```

Comment:
`TEMPLATE_PATH` should link to `resources` directory where `sorter.fthl` locates, not the temporary directory.  
On the other hand, `GENERATING_PATH` should link to temporary directory and I think we should try to use the one that Flink use.